wlauto/common/android/device.py

```diff
@@ -589,7 +589,7 @@ def perform_unlock_swipe(self):
             start = 100
             stop = width - start
             self.execute(command.format(start, swipe_heigh, stop, swipe_heigh))
-        if self.swipe_to_unlock == "vertical":
```

Comment:
This is a bug fix. Shouldn't this be an either or? Anyway I will remove it from this commit.